modules/admin-ui/src/main/webapp/scripts/shared/partials/modals/event-details.html

```diff
@@ -1483,31 +1486,46 @@ <h3>{{ item.name }}</h3>
             <header translate="EVENTS.EVENTS.DETAILS.WORKFLOW_DETAILS.TITLE"><!-- Workflow Details --></header>
             <div class="obj-container">
               <table class="main-tbl">
-                <tr>
-                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.ID"><!-- ID --></td>
```

Comment:
I would argue that this is more or less the most useful information in this view. Everything else, you have either seen before or is not really interesting at all. Hence I would prefer to leave this at the top.

Reply:
What exactly is useful about the workflow identifier? The only thing I could think of is using the identifier for REST calls, but that is developer/system administration stuff... For any other uses, the identifier is meaningless.

Reply:
Right, but there's no other easy way to get that information from the UI...

Reply:
Re-introduced this column.